backend/src/nodes/impl/external_stable_diffusion.py

```diff
@@ -58,9 +58,14 @@ class ExternalServiceTimeout(Exception):
     pass
 
 
-def get(url) -> Dict:
+def get(url, timeout: float = STABLE_DIFFUSION_REQUEST_TIMEOUT) -> Dict:
     try:
-        response = requests.get(url, timeout=STABLE_DIFFUSION_REQUEST_TIMEOUT)
+        response = requests.get(
+            url,
+            timeout=timeout
+            if timeout is not None
+            else STABLE_DIFFUSION_REQUEST_TIMEOUT,
+        )
```

Comment:
I find it worrying that pyright accepted this code. It really should have complained because it knows that `timeout` can never be `None`.

```suggestion
        response = requests.get(url, timeout=timeout)
```